touchforms/formplayer/api.py

```diff
@@ -293,7 +294,7 @@ def post_data_helper(d, auth, content_type, url, log=False):
 
 
 def formplayer_post_data_helper(d, auth, content_type, url):
-    d['nav_mode'] = 'prompt'
+    d = get_formplayer_session_data(d)
```

Comment:
it looks like `perform_experiment` calls `get_candidate_session_data` before calling `formplayer_post_data_helper` so it seems like calling `get_formplayer_session_data` is unnecessary in that flow.

should this call to `get_formplayer_session_data` be moved to be directly under the toggle check?